content/blog/2023-08-13-hashicorp-bsl/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "The Statement for SkyWalking users on HashiCorp license changes"
+date: 2023-08-13
+author: "Apache SkyWalking"
+description: "HashiCorp announced to adopt the Business Source License from Mozilla Public License v2.0 (MPL 2.0). SkyWalking community posts the statement for our own users to explain the evaluation of the potential implications."
+startTime: 2023-08-13T00:00:00Z
+endTime: 2023-09-30T10:00:00Z
+---
+
+On Aug. 10th, 2023, HashiCorp announced to adopt the Business Source License (BSL) from Mozilla Public License v2.0 (MPL 2.0), here is their [post](https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license). 
+They officially annouced they have changed the license for the **ALL** of their open-source products from the previous MPL 2.0 to a source-available license, BSL 1.1. 
+Meanwhile, HashiCorp APIs, SDKs, and almost all other libraries will remain MPL 2.0.
+
+HashiCorp Inc. is one of the most important vendors in the cloud-native landscape, as well as Golang ecosystem. This kind of changes would have potential implications for 
+SkyWalking, which is closely integrated with cloud-native technology stacks. 
+
+
+# Conclusion First
+- **What does that mean for SkyWalking users?**
+
+SkyWalking community has evaluated our dependencies from HashiCorp products and libraries, the current conclusion is
+
+**SkyWalking users would NOT suffer any implication. All components of SkyWalking don't have hard-dependency on BSL license affected codes.**
+
+SkyWalking community have found out all following dependencies of all relative repositories, all licenses are TRUELY stayed unchanged, and compatible with Apache 2.0 License.
+
+- [x] OAP Server @kezhenxu94 @wu-sheng 
+  - consul-client Apache 2.0 Repo archived on Jul 27, 2023
```

Comment:
This is actually a third party lib, not from HashiCorp

Reply:
Yes, it is. But HashiCorp owns the APIs.